pkg/kubelet/kubelet.go

```diff
@@ -2431,6 +2437,47 @@ func (kl *Kubelet) fastStatusUpdateOnce() {
 	}
 }
 
+// CheckpointContainer tries to checkpoint a container. The parameters are used to
+// look up the specified container. If the container specified by the given parameters
+// cannot be found an error is returned. If the container is found the container
+// engine will be asked to checkpoint the given container into the kubelet's default
+// checkpoint directory.
+func (kl *Kubelet) CheckpointContainer(
```

Comment:
Missing unit test? - Should be relatively easy to add one though.

Reply:
Unit test added.